app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -58,6 +60,27 @@ class SyncIntegrationTest {
         bookmarkAfterSyncIsShown()
     }
 
+    @Test
+    fun checkAccountSettings() {
+        signInFxSync()
+        mDevice.waitNotNull(Until.findObjects(By.text("Settings")), TestAssetHelper.waitingTime)
+
+        goToAccountSettings()
+        // This function to be added to the robot once the status of checkboxes can be checked
```

Comment:
can we reference an issue number here in the comment?

Reply:
Right, I have to open an issue so that IDs or some kind of identifiers are added to those checkboxes.